recipes/r-metaboanalyst/meta.yaml

```diff
@@ -0,0 +1,138 @@
+{% set version = '2.0.0' %}
+
+package:
+  name: r-metaboanalyst
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url: https://github.com/xia-lab/MetaboAnalystR/releases/download/v{{ version }}/MetaboAnalystR_{{ version }}.tar.gz
+  sha256: 00656db1f854483f28c842c54621a1a0f79866c468baa070784ca6526884032f
+
+
+build:
+  noarch: generic
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  host:
+    - r-base
+    - r-tidyverse
+    - bioconductor-keggrest
+    - r-rserve
+    - r-ellipse
+    - r-scatterplot3d
+    - r-cairo
+    - r-randomforest
+    - r-catools
+    - r-e1071
+    - r-som
+    - bioconductor-impute
+    - bioconductor-pcamethods
+    - r-rjsonio
+    - r-rocr
+    - bioconductor-globaltest
+    - bioconductor-globalancova
+    - bioconductor-rgraphviz
+    - bioconductor-preprocesscore
+    - bioconductor-genefilter
+    - r-pheatmap
+    - bioconductor-sspa
+    - bioconductor-sva
+    - r-rcpp
+    - r-proc
+    - r-data.table
+    - bioconductor-limma
+    - r-car
+    - r-fitdistrplus
+    - r-lars
+    - r-hmisc
+    - r-magrittr
+    - r-xtable
+    - r-pls
+    - r-caret
+    - r-lattice
+    - r-igraph
+    - r-gplots
+    - bioconductor-kegggraph
+    - r-reshape
+    - r-rcolorbrewer
+    - r-tibble
+    - bioconductor-siggenes
+    - r-plotly
+    - bioconductor-xcms
+    - bioconductor-camera
+    - bioconductor-fgsea
+    - bioconductor-msnbase
+    - bioconductor-biocparallel
+    - r-metap
+    - r-reshape2
+    - r-scales
+    - r-devtools
+    - r-spls
+
+  run:
+    - r-base
+    - r-tidyverse
+    - bioconductor-keggrest
+    - r-rserve
+    - r-ellipse
+    - r-scatterplot3d
+    - r-cairo
+    - r-randomforest
+    - r-catools
+    - r-e1071
+    - r-som
+    - bioconductor-impute
+    - bioconductor-pcamethods
+    - r-rjsonio
+    - r-rocr
+    - bioconductor-globaltest
+    - bioconductor-globalancova
+    - bioconductor-rgraphviz
+    - bioconductor-preprocesscore
+    - bioconductor-genefilter
+    - r-pheatmap
+    - bioconductor-sspa
+    - bioconductor-sva
+    - r-rcpp
+    - r-proc
+    - r-data.table
+    - bioconductor-limma
+    - r-car
+    - r-fitdistrplus
+    - r-lars
+    - r-hmisc
+    - r-magrittr
+    - r-xtable
+    - r-pls
+    - r-caret
+    - r-lattice
+    - r-igraph
+    - r-gplots
+    - bioconductor-kegggraph
+    - r-reshape
+    - r-rcolorbrewer
+    - r-tibble
+    - bioconductor-siggenes
+    - r-plotly
+    - bioconductor-xcms
+    - bioconductor-camera
+    - bioconductor-fgsea
+    - bioconductor-msnbase
+    - bioconductor-biocparallel
+    - r-metap
+    - r-reshape2
+    - r-scales
+    - r-spls
+
+test:
+  commands:
+    - $R -e "library('MetaboAnalystR')"           # [not win]
+
+about:
+  home: https://github.com/xia-lab/MetaboAnalystR
+  license: GPL-3.0
+  summary: MetaboAnalystR 2.0 contains the R functions and libraries underlying the popular MetaboAnalyst web server.
```

Comment:
@Slugger70 can you please include a license_file ... we need to do this for GPL packages.